src/index.html

```diff
@@ -9,6 +9,30 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Enclosures</h1>
+  <div class="block-blue" id="qa-block">
+    <div class="block-orange">
+      <p class="block-text">In programming languages, a closure is a technique
+         for implementing lexically scoped name binding in a language with
+         first-class functions. Operationally, a closure is a record storing a
```

Comment:
when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear